pkg/networkmanager/index.html

```diff
@@ -352,59 +352,60 @@ <h4 class="pf-c-alert__title" translate="yes">Network devices and graphs require
   </script>
 
   <div id="network-interface" class="pf-c-page" hidden>
-    <main  class="pf-c-page__main" tabIndex="-1">
+    <main class="pf-c-page__main" tabindex="-1">
       <section class="pf-c-page__main-breadcrumb pf-m-light">
-          <nav class="pf-c-breadcrumb" aria-label="breadcrumb">
-              <ol class="pf-c-breadcrumb__list">
-                <li class="pf-c-breadcrumb__item"><a class="pf-c-breadcrumb__link" role="link" tabindex="0" translate="yes">Networking</a></li>
-                <li class="pf-c-breadcrumb__item">
-                    <span class="pf-c-breadcrumb__item-divider">
-                        <i class="fa fa-angle-right" aria-hidden="true"></i>
-                    </span>
-                    <a class="pf-c-breadcrumb__link pf-m-current"></a>
-                </li>
-              </ol>
-          </nav>
+        <nav class="pf-c-breadcrumb" aria-label="breadcrumb">
+          <ol class="pf-c-breadcrumb__list">
+            <li class="pf-c-breadcrumb__item"><a class="pf-c-breadcrumb__link" role="link" tabindex="0" translate="yes">Networking</a></li>
+            <li class="pf-c-breadcrumb__item">
+              <span class="pf-c-breadcrumb__item-divider">
+                <i class="fa fa-angle-right" aria-hidden="true"></i>
+              </span>
+              <a class="pf-c-breadcrumb__link pf-m-current"></a>
+            </li>
+          </ol>
+        </nav>
       </section>
-      <section class="pf-c-page__main-section pf-m-light">
+
+      <section class="networking-graphs networking-graphs-interface pf-c-page__main-section pf-m-light">
         <div id="network-interface-graphs-element"></div>
       </section>
+
       <section class="pf-c-page__main-section">
         <div class="pf-l-gallery pf-m-gutter">
-            <div class="pf-c-card">
-              <div class="pf-c-card__header">
-                <div class="pf-c-card__title">
-                  <span id="network-interface-name"></span>
-                  <span id="network-interface-hw"></span>
-                  <span id="network-interface-mac"></span>
-                </div>
-                <div class="pf-c-card__actions">
-                  <button class="pf-c-button pf-m-danger network-privileged" id="network-interface-delete" translate="yes">Delete</button>
-                  <span id="network-interface-delete-switch" class="network-privileged">
-                  </span>
-                </div>
+          <div class="pf-c-card network-interface-details">
```

Comment:
All these whitespace changes will conflict with the Reactification PR :/ Which moved most of all this html to React.

Reply:
Ugh, sorry.

Can you revert the whitespace change in your PR and then squash?

As long as you have the commit still tracked, this should work:

`git revert cdd5210772b9b923ded93d899ebd8f07d473c4dc`

(Of course, this would need to be before your changes on it. You could do that in another branch rebase yours on top. It should then cleanly apply. Then you could squash it.)

Reply:
Oh, I forgot that there are some legit changes. https://github.com/cockpit-project/cockpit/pull/15054/commits/cdd5210772b9b923ded93d899ebd8f07d473c4dc#diff-7c4ae9e02bd2dee50641ed1eb72a6bd556045b341d0c2104df8ce300ae940faaR383-R385 Whoops.

But it appears that's it.

Reply:
Nope, GitHub isn't showing the diff correctly. There are additional changes.

I guess git diff's -w might help. But then this starts to become more effort than resolving the conflicts, I suppose.